recipe/meta.yaml

```diff
@@ -1,18 +1,18 @@
-{% set version = "5.1.7" %}
+{% set version = "5.1.9" %}
 
 package:
   name: giflib
   version: {{ version }}
 
 source:
   url: http://downloads.sourceforge.net/project/giflib/giflib-{{ version }}.tar.gz
-  sha256: 94096e4f43fd06071bbb4a6827a9ab4eeff52036adb662f329ffc7c1d198436d
+  sha256: 36ccab06aa43e5d608cdd74902f89c47fd55c348009798434ba5798967454057
   patches:
     - makefile.patch
 
 build:
-  number: 1
-  skip: True  # [win]
+  number: 0
+  skip: True  # [win or osx]
```

Comment:
giflib's new makefile needs some heavy patching to build on anything but Linux.
I'm not in the mood to maintain those patches so I'm just skipping the builds that are too hard at the moment.